### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by milosvukadinovic**

### CODE REVIEW

File: src/Components/Display.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const display = () => (
+  <div>
+    <h1>Display</h1>
```
Comment (SunnyAsar) [inline src/Components/Display.js]: display needs to accept as props the result of the calculation

File: src/Components/Display.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  </div>
+);
+
```
Comment (SunnyAsar) [inline src/Components/Display.js:9]: also lets add a default prop value for when there is no result set yet and let have propTypes as well.

File: src/Components/Button.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const button = (prop) => (
+  <div>
+    <h2>{prop.name}</h2>
+  </div>
+);
```
Comment (SunnyAsar) [inline src/Components/Button.js:9]: let's add propType here too to validate that prop.name is a string.

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hello @milosvukadinovic  👋 
Nice work setting up your project neatly, just missed out on a few validation of the propType and setting default values, i have commented below what needs change.
#### Note
- There is a milestone for styling but setting it up now to at least be a grid wont hurt
- Clean up your `ReadMe` and make it descriptive of the project.
- You have some linting issues, please fix them and setup a [linter](https://github.com/microverseinc/linters-config/tree/master/react-redux) for your repository too.

**Please make the changes and request another review when you are done**
Good job so far

**Review by BalintMendli — COMMENTED** [review]
Hi @milosvukadinovic ,

I see that you have this milestone left over.

As requirements all met I mark it approved now.

👍

**Pull request merged**